jerry-core/lit/lit-magic-strings.h

```diff
@@ -34,6 +34,11 @@ typedef enum
 #include "lit-magic-strings.inc.h"
 #undef LIT_MAGIC_STRING_DEF
 #undef LIT_MAGIC_STRING_FIRST_STRING_WITH_SIZE
+  LIT_REAL_MAGIC_STRING__COUNT, /**< number of real magic strings */
```

Comment:
Not a real issue, but the phase "real magic strings" is ambiguous to me. I'd add more description to this or rename to something else, like `common`, `general` or ` standard magic strings`